<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompositeTileSourceConfigurationWidget</class>
 <widget class="QWidget" name="CompositeTileSourceConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="markPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>439</width>
        <height>315</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Marks</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>81</width>
         <height>251</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <property name="rowWrapPolicy">
         <enum>QFormLayout::DontWrapRows</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>80</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="addMark_button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/images/edit_add.png</normaloff>:/icons/images/edit_add.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="removeMark_button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/images/editdelete.png</normaloff>:/icons/images/editdelete.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="changeColor_button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/images/palette.png</normaloff>:/icons/images/palette.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="moveMark_button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/images/drag.png</normaloff>:/icons/images/drag.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>80</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="routePage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>439</width>
        <height>315</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Routes</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>77</width>
         <height>251</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <property name="rowWrapPolicy">
         <enum>QFormLayout::DontWrapRows</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>80</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="addRoute_button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/images/edit_add.png</normaloff>:/icons/images/edit_add.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="removeRoute_button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/images/editdelete.png</normaloff>:/icons/images/editdelete.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="createRoute_button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/images/slpine.png</normaloff>:/icons/images/slpine.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="linkRoute_button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/images/car_object2.png</normaloff>:/icons/images/car_object2.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>80</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="geoPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>439</width>
        <height>315</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Geo Objects</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>73</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item row="0" column="1">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>80</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QToolButton" name="addGetObject_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/images/edit_add.png</normaloff>:/icons/images/edit_add.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="removeGeoObject_button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/images/editdelete.png</normaloff>:/icons/images/editdelete.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>80</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QToolButton" name="addGeoObject_toolButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/images/edit_add.png</normaloff>:/icons/images/edit_add.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
